e2e/src/app.e2e-spec.ts

```diff
@@ -1,5 +1,7 @@
-import { AppPage } from './app.po';
-import { browser, logging } from 'protractor';
+/* eslint-disable @typescript-eslint/no-floating-promises */
+
+import {AppPage} from './app.po';
+import {browser, by, logging} from 'protractor';
 
 describe('workspace-project App', () => {
   let page: AppPage; // eslint-disable-line @typescript-eslint/no-unused-vars
```

Comment:
remove this linter disabling

Reply:
please fix this

Reply:
why have you marked it as solved while it is still there?

Reply:
`/* eslint-disable @typescript-eslint/no-floating-promises */` has been already been removed.
Should I remove `// eslint-disable-line @typescript-eslint/no-unused-vars` too ? I haven't added that one.

Reply:
You were improving these tests, so that it is still your responsibility. And actually this was a "disabling" which was there because the variable was not used (but I didn't want to use this code "skeleton") now you use it, so it makes sense to remove it.

And as I asked (twice), that's probably that I wanted to be done 😉.

Reply:
fixed

Reply:
but it is still there  🤨

Reply:
didn't pushed... sorry. Now it should be fixed.